pizza.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Pizza-order</title>
+    </head>
+    <body>
+        <header>
+            <h2>Pizza Order</h2>
+        </header>
+        <div><form action="https://www.google.ro/">
+            <label for="nume"><b>Your Name:</b></label>
+            <input type="text" name="nume" minlength="3" maxlength="10" placeholder="your name" required><br><br>
+            <label for="email"><b>Your email:</b></label>
+            <input type="email" name="email" placeholder="your email" required><br><br>
+            <label for="topping"><b>Topping:</b></label>
+            <input type="radio" name="topping" value="Supreme"><b>Supreme</b>
+            <input type="radio" name="topping" value="Vegetarian" checked ><b>Vegetarian</b>
+            <input type="radio" name="topping" value="Hawaian"><b>Hawaian</b><br><br>
```

Comment:
Fiecare input este legat de un label. Acest lucru se face prin atributele for si id dupa cum am sugerat mai jos.
```suggestion
            <p><b>Topping:</b></p>
            <input type="radio" id="supreme" name="topping" value="Supreme"><label for="supreme"><b>Supreme</b></label>
            <input type="radio" id="vegetarian" name="topping" value="Vegetarian" checked ><label for="vegetarian"><b>Vegetarian</b></label>
            <input type="radio" name="topping" value="Hawaian"><label for="hawaian"><b>Hawaian</b></label><br><br>
```